js/main.js

```diff
@@ -29,6 +36,9 @@ $return.addEventListener('click', e => {
   while ($brCityList.firstChild) {
     $brCityList.removeChild($brCityList.firstChild);
   }
+  while ($favesList.firstChild) {
+    $favesList.removeChild($favesList.firstChild);
+  }
```

Comment:
Instead of doing all these `while` loops, I think there's a better way of accomplishing this. Check out this method https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren.

I think you can just do something like this `$favesList.replaceChildren();` and it'll remove all children nodes. That way you don't have 4 separate `while` loops